app/harvesters/open_alex/open_alex_references_converter.py

```diff
@@ -0,0 +1,129 @@
+from typing import AsyncGenerator
+from app.db.daos.contributor_dao import ContributorDAO
+from app.db.models.concept import Concept
+
+from app.db.models.contribution import Contribution
+from app.db.models.contributor import Contributor
+from app.db.models.title import Title
+from app.db.models.reference import Reference
+from app.db.models.abstract import Abstract
+from app.db.session import async_session
+
+from app.harvesters.abstract_references_converter import AbstractReferencesConverter
+from app.harvesters.json_harvester_raw_result import JsonHarvesterRawResult
+from app.harvesters.open_alex.open_alex_document_type_converter import (
+    OpenAlexDocumentTypeConverter,
+)
+
+
+class OpenAlexReferencesConverter(AbstractReferencesConverter):
+    """
+    Converts raw data from OpenAlex to a normalised Reference object
+    """
+
+    async def convert(self, raw_data: JsonHarvesterRawResult) -> Reference:
+        """
+        Convert raw data from OpenAlex to a normalised Reference object
+        :param raw_data: raw data from OpenAlex
+        :return: Reference object
+        """
+        json_payload = raw_data.payload
+        new_ref = Reference()
+
+        language = json_payload.get("language", None)
+        new_ref.titles.append(self._title(json_payload, language))
```

Comment:
Same error with ORCID 0000-0001-9603-7681

Reply:
One of his document: https://openalex.org/works/W2069786235 has an asbtract of lenght 2705. Causing: 
> HINT:  Values larger than 1/3 of a buffer page cannot be indexed.

Problem is discussed in #230